zerver/lib/streams.py

```diff
@@ -1132,3 +1125,14 @@ def get_subscribed_private_streams_for_user(user_profile: UserProfile) -> QueryS
         .filter(subscribed=True)
     )
     return subscribed_private_streams
+
+
+def send_stream_deletion_event(realm: Realm, users: Iterable[int], streams: list[Stream]) -> None:
```

Comment:
Would be good to name `users` variable as `user_ids`. And we can probably just pass stream IDs here instead of stream obejcts.

Reply:
I think stream objects is better, it would avoid writing `[stream.id for stream in streams]` every time we pass this argument from the caller

Reply:
Ok. Sounds good.